src/main/java/com/cloudbees/plugins/credentials/CredentialsProvider.java

```diff
@@ -836,4 +807,12 @@ public void remove() {
         return null;
     }
 
+    public static ExtensionList<CredentialsProvider> all() {
+        // TODO switch to ExtensionList.lookup once Jenkins 1.572+ is the baseline
+        final Jenkins jenkins = Jenkins.getInstance();
+        return jenkins == null
+                ? ExtensionList.create((Jenkins)null, CredentialsProvider.class)
```

Comment:
Because there is a second method that has the `Hudson` as the first parameter so when using a `null` you need to tell the compiler the type to use... we could use the `jenkins` variable but that is a known `null` so code analysis complains about that too